### DISCUSSION TITLE
how to use lammps in deepmd-docker

### DISCUSSION THREAD

**Discussion by wjinjin** [deepmodeling/deepmd-kit#25] [Q&A]
We have installed deepmd with the docker project and got the graph.pb file. However, we don't know how to run MD with Lammps in docker?

**Comment by amcadmus** [comment]
@frankhan91   Does one need to install lammps manually in docker?

**Comment by frankhan91** [comment]
LAMMPS should be installed in docker. Once you have graph.pb file with the frozen model, you can follow the [instruction](https://github.com/deepmodeling/deepmd-kit#run-md-with-lammps) to run MD with LAMMPS.

**Comment by wjinjin** [comment]
@frankhan91 Thank you for your answer. To run MD in LAMMPS with graph.pb file, LAMMPS's deepmd-kit module must be installed in docker. However, I have not found any guide on installing LAMMPS's deepmd-kit module in docker.

**Comment by frankhan91** [comment]
LAMMPS should already be installed if you build docker successfully. Can you take a look at line 57-59 in Dockerfile or the log of building?

**Comment by wjinjin** [comment]
Thank you for your replies. I have reinstalled lammps according to the Dockerfile.
